src/client/Microsoft.Identity.Client/Platforms/Features/OpenTelemetry/OtelInstrumentation.cs

```diff
@@ -168,51 +171,55 @@ public void LogSuccessMetrics(
                 new(TelemetryConstants.Platform, platform),
                 new(TelemetryConstants.ApiId, apiId),
                 new(TelemetryConstants.TokenSource, authResultMetadata.TokenSource),
-                new(TelemetryConstants.CacheLevel, authResultMetadata.CacheLevel));
+                new(TelemetryConstants.CacheLevel, authResultMetadata.CacheLevel),
+                new(TelemetryConstants.TokenType, authResultMetadata.TelemetryTokenType));
             }
         }
 
-        public void IncrementSuccessCounter(string platform, 
-            ApiEvent.ApiIds apiId, 
+        public void IncrementSuccessCounter(string platform,
+            ApiEvent.ApiIds apiId,
             string callerSdkId,
             string callerSdkVersion,
             TokenSource tokenSource,
-            CacheRefreshReason cacheRefreshReason, 
+            CacheRefreshReason cacheRefreshReason,
             CacheLevel cacheLevel,
-            ILoggerAdapter logger)
+            ILoggerAdapter logger,
+            int tokenType)
         {
             if (s_successCounter.Value.Enabled)
             {
                 s_successCounter.Value.Add(1,
                         new(TelemetryConstants.MsalVersion, MsalIdHelper.GetMsalVersion()),
                         new(TelemetryConstants.Platform, platform),
                         new(TelemetryConstants.ApiId, apiId),
-                        new(TelemetryConstants.CallerSdkId, callerSdkId ?? string.Empty),
-                        new(TelemetryConstants.CallerSdkVersion, callerSdkVersion ?? string.Empty),
+                        new(TelemetryConstants.CallerSdkId, callerSdkId ?? string.Empty + " " + callerSdkVersion ?? string.Empty),
```

Comment:
Can this be `'` separated instead of space

Reply:
sure